TS.fsx

```diff
@@ -381,8 +381,15 @@ let EmitNamedConstructors () =
 
 let EmitInterfaceDeclaration (i:Browser.Interface) =
```

Comment:
There are no classes in `dom.generated.d.ts`, all types in the spec translate to an interface (instance side) and a declared var constructor function (static side).